### ISSUE TITLE
Discussion of Brunner 1993, Crane 2002, Terras 2010

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2020-2021#1]
Read and discuss:
* Brunner, Theodore F. (1993), “Classics and the Computer: The History of a Relationship.” In _Accessing Antiquity_, ed. Jon Solomon, pp. 10-33. Tucson: U. Arizona Press.
* Crane, Gregory (2004). "Classics and the Computer: An End of the History." In _A Companion to Digital Humanities_, eds. Schreibman, Siemens, Unsworth, pp. 46-55. Oxford: Blackwell. Available: http://www.digitalhumanities.org/companion/
* Terras, Melissa (2010), “The Digital Classicist: Disciplinary Focus and Interdisciplinary Vision.” In _Digital Research in the Study of Classical Antiquity_, eds. Bodard, Mahony, pp. 171-189. Ashgate. Available: http://www.ucl.ac.uk/infostudies/melissa-terras/research/Chapter_10_Terras.pdf

Questions to get us started:
1. What is Digital Classics?
1. Does Digital Classics have different needs and goals from Digital Humanities?
1. Would the answer be different if you were a philologist vs. historian vs. archaeologist?
1. Which DH methods do you think help us to ask new questions of the ancient world?

**Comment by nicolealexandra33** [comment]
[Brunner 1993.docx](https://github.com/SunoikisisDC/SunoikisisDC-2020-2021/files/5339831/Brunner.1993.docx)

**Comment by gabrielbodard** [comment]
@nicolealexandra33 it might be easier to follow if you just pasted the content of your summary into the comment, rather than attaching a doc. (You could also learn a bit about Markdown to format the title etc. if you like!)

**Comment by despinaborcea** [comment]
**Terras, Melissa (2010), "The Digital Classicist: Disciplinary Focus and Interdisciplinary Vision." In Digital Research in the Study of Classical Antiquity, eds. Bodard, Mahony, pp. 171-189. Ashgate.**
[http://www.ucl.ac.uk/infostudies/melissa-terras/research/Chapter_10_Terras.pdf](url)

In this article, Terras explores the advantages and disadvantages of an interdisciplinary approach to the Greek and Roman world in the form of Digital Classics. After a brief discussion on the issue of generally defining a discipline and its boundaries, the historical development of the Digital Classicist is introduced, with examples varying from _Ibycus_ to the _Perseus Project_. 

Firstly, the benefits of an interdisciplinary approach fall under the umbrella of accessibility: not only are ancient texts available online, but also their critical apparatus, explanatory notes etc., facilitating the scholars’ access to a more straightforward and enriching study experience. To illustrate the non-restrictive scope of such an approach, Terras selects two different digital projects, in Classics and Archaeology, respectively. The first is a system called eSAD used for elucidating the texts of the Vindolanda tablets through imagery, as well as to propose different options to scholars’ understanding of these texts. A different example is VERA (Virtual Environments for Research in Archaeology), which enables archaeologists, through a digital database, to (re)visit the excavations in Silchester Roman Town in Hampshire. 

Nevertheless, the interdisciplinary approach retains some drawbacks. Firstly, due to the major differences between Classics and Computer Science, Classical scholars may experience difficulties in appropriating notions of the other field. Moreover, there can be clashes in interdisciplinary publications, as scholars of both fields may claim merit over them. Unhelpful leadership that does not foster communication between the two fields can also negatively affect the project and finally, securing funding can prove difficult.

**Comment by nicolealexandra33** [comment]
**Brunner, Theodore F. (1993), “Classics and the Computer: The History of a Relationship.” In Accessing Antiquity, ed. Jon Solomon, pp. 10-33. Tucson: U. Arizona Press.Brunner, Theodore F. (1993), “Classics and the Computer: The History of a Relationship.” 	In Accessing Antiquity, ed. Jon Solomon, pp. 10-33. Tucson: U. Arizona Press.**

This article describes the evolution of digital classics in its first few decades, beginning in the 1960s with the digitisation and data entry of the works of Saint Thomas Aquinas to the creation of an affordable CD-ROM in the 1980s which contained large amounts of Greek texts. Despite the argument for using newer technologies such as computers in the study of classics since 1949, it was not until 1987 when the American Philological Association (APA) declared that the digitisation of ancient texts as well as the use of other computer software were invaluable tools to study classics. Based on the scope of this article, the definition of digital classics comprises mainly of the digitisation of ancient texts, thus focusing primarily on the goals and needs of classical philologists, as it does not include some of the computer-related breakthroughs in other fields such as archaeology (GIS, 3D reconstruction, etc.). Two major examples of this are given: (1) in 1969, a newly established Summer Institute in Computer Applications to Classical Studies, endorsed by the APA, began the ‘first centrally located corpus of Greek and Latin texts and other materials’; and (2), in 1972, the creation of the Thesaurus Linguae Graecae (TLG) for ‘texts of ancient Greek literature in machine-readable form’. Brunner also summarises some issues that arose during the progression of these projects, including the ongoing tensions between the TLG and the Repository under the jurisdiction of the APA, as well as the early lack of standardisation of verified and corrected texts published by the TLG. Lack of accessibility and funding of TLG texts in its early years meant that these texts were not freely accessible, thus limiting its use to the greater classics community.

**Comment by gabrielbodard** [comment]
Another question to think about (for all readings):

* Who is the author, and what is their connection to the material/project(s) they're describing in this paper? Where was the paper published and does that have any impact on the content, intent, agenda?

**Comment by annaszajbely** [comment]
Greg Crane: Classics and the Computer: An End of the History

This article was a summary about the early usage of IT technology in the field of Classics. 
One hardship of “classicsal informatics” in the 1970s was that it was obviously not a priority. Academic technology specialists within the higher education system devoted 1 percent of the gained funding for the IT development in the field of humanities, for the major priority was science and medicine. “The question is not whether they can create a classical informatics but whether such broad rubrics as computational humanities, computing in the humanities, cultural informatics, and so on, are sufficient or whether they should more aggressively strive to situate themselves within an informatics that covers the academy as a whole.”
At the beginning of the 1970s projects had been started to react to the issues of classical computing infrastructure; such as the Thesaurus Linguae Graecae (TLG), Database of Classical Bibliography, the Bryn Mawr Classical Review, the Duke Databank of Documentary Papyri, the Perseus Project, and David’ Packard’s Ibycus system. Later on, in the 1980s, the appearance of the Unix operating system and the C programing language provided more opportunities fro development, which are continuous in the present day. MS DOS and the IBM PC which debuted in 1981 and Macinthos appeared in 1984. As a result of the development the problems of speed, disk storage (which was most crucial for classicists, and prize of both the hardwares and softwares started to solve.

**Comment by chiaradimaio** [comment]
I would start with the consideration that ‘non progredi est regredi’: of course, this applies to any field of study, for which technological development has proved a huge help and simplification. The conscious use of informatic tools allows researchers to broaden their view, to access, store and compare data in a faster and easier way. I could not agree more with Melissa Terras, saying that ‘Classics is no different from any other academic subject in this regard’. 
Digital Classics should not be regarded as a ‘new’ discipline, but as a naturally evolutionary approach towards a discipline. The background of a Classicist in the 21th Century necessarily has to be different from the past, as Greg Crane addressed from the early ‘70s. 
I think that one of the core points of the discussion, that Brunner observed in his article in the section about the ‘80s, is still crucial in 2020: there is a ‘gap in understanding between those of us who were new to the technology and our technological colleagues who were innocent of classics’. This consideration can be daunting, because the conservative approach of many classical scholars (that can be seen, to a certain extent, as a natural heritage of their bond with antiquity) leads, too often, to a shift between a ‘progressive’ Academia and a more ‘traditional’ view on Classics. Opening to an inter-disciplinary vision has always been crucial for developing new knowledge. Furthermore, it is essential to consider that even the most backward-looking scholar, nowadays would not be able to operate without the tools that he has been offered by the ‘allies of other disciplines’ (Terras).
I am approaching Digital Classics for the first time and I think that its opportunities and definition will become clearer to me as soon as I get more into its practical aspect.

**Comment by nicolealexandra33** [comment]
Q2- Crane argues that ‘A tiny field such as classics, operating at the margins of the humanities, cannot afford a distinctive and autonomous history of its own. For classicists to make successful use of information technology, they must insinuate themselves within larger groups, making allies of other disciplines and sharing infrastructure’. In other words, he argues (as well as Terras) that in order to maximise the use of new technologies in the study of classics, classists need to become more holistic in their approach as well as more inter-disciplinary with regard to their research questions.

Qs 2 and 3- One issue that particularly affects Digital Classists and Philologists would be the need for searching, producing, and translating Greek scripts efficiently.
Terras explains that all humanities disciplines benefit from and seek new technologies which facilitate communication. 

Q 5- Brunner, the author of one of the articles, was also one of the founders of the TLG which explains his in-depth knowledge of all of the issues that the TLG faced early in its creation. It also explains his focus on Digital Classics and Philology specifically rather than the Digital Humanities.

**Comment by ghost** [comment]
Q2: Digital Classics has a more defined focus, I believe, than digital humanities. Digital classics requires the translation of ancient Greek, and the ability to search for and read scrolls, tablets, papyri, or other artifacts in a quicker and more efficient manner. Digital Humanities seems like a broader topic, while Digital Classics is more specific and focused on what they require.

Q3: As someone who has studied Classical art history and archaeology, the ability to quickly research a word or phrase in ancient Greek is a necessary one, more so in art history. Archaeology depends upon machines such as ground-penetrating radar, Lidar scanning, and geographical information science which, depending on the research, may or may not be included in "Digital Classics." It is a different focus than what is described in the articles. When the authors discussed digital classics, they focused on papyri and ancient Greek translations. Therefore, I am unclear on whether the technology used for archaeology would be included in digital classics/digital humanities.

Q4: The eSAD (e-science and ancient documents) project in Terras' article would allow new questions to arise because artifacts that have been damaged and could not be read previously can now be deciphered due to innovative image processing algorithms; allowing researchers to ask even more questions or to have some questions answered.

**Comment by gabrielbodard** [comment]
@HLBallard re your comment on Q3: the only context in which geo- and similar archaeological technologies are not considered to be digital humanities or digital classics, is when archaeologists argue that these are a core part of their discipline, and therefore do not belong to a separate "digital" subdiscipline (much as we said about not needing a "digital physics" label last week). One could make the same argument for many of the digital methods we talk about in classics, of course, e.g. search large text corpora; they are fundamental to everyone's work in philology now. I personally would include in digital classics any area where digital development is being performed by people with classical interests as well as computer science tools simply being used as a matter of course (including therefore GIS etc.)—as opposed to wordprocessing, say, which is not developed _by_ classicists because we have no unique needs.
